woocommerce/woocommerce
# [Experimental] Price filter: fix range slider update

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes issue with the price filter where the range slider control is not updating the range bar and value inputs.

| Before | After |
|--------|--------|
| https://github.com/user-attachments/assets/1ba4f812-4bba-4106-9365-824a4460a03a | https://github.com/user-attachments/assets/9cc3864c-0cb5-455e-86ef-06472c5e471f | 


### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

**Test by developers only**

Prereq:

Ensure you have the WC Beta Tester Plugin installed and go to Tools > WCA Test Helper > Features. Toggle on the experimental-blocks option.

1. Go to the editor and navigate to Product Catalog template.
2. Add the Product Filters (Experimental) block to the editor.
3. Save and go to the frontend default shop page.
4. Move the minimum and maximum slider around. Ensure you see the range bar moving per design and also you see the value input changing while you're moving the slider.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

Price slider: fix range bar styling and dynamic input
</details>
